app/adapters/alma_adapter/status.rb

```diff
@@ -14,6 +14,10 @@ def initialize(bib:, holding_item_data:, holding:)
   end
 
   def to_s
-    return "On-Site" if holding["availability"] == "available"
+    return "Some items not available" if holding["availability"] == "check_holdings"
+    return holding["availability"].titlecase if holding["availability"]
+
+    # For electronic holdings
+    return holding["activation_status"].titlecase if holding["activation_status"]
```

Comment:
I'm not sure we are aware of this use case from the orangelight side; we might need more info.

Reply:
This is the reason I have been hesitant to close the two tickets about availability (single bib and multi-bib). 

There are new cases that Alma brings that we didn't have to deal with in Voyager. Hence the note in the code: https://github.com/pulibrary/bibdata/blob/alma/app/adapters/alma_adapter/availability_status.rb#L28-L64.

Reply:
Okay let's talk about it at an alma checkin.